cri.go

```diff
@@ -127,6 +135,37 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 	return s, nil
 }
 
+// getServices get service instances from plugin context.
+func getServices(ic *plugin.InitContext) (map[string]interface{}, error) {
```

Comment:
Could this function just return the service options? This would prevent haven't to handle the list of services in 2 places

Reply:
Let me see. Need to a table to map the service to corresponding option and type.

Reply:
Done.